src/_includes/navbar.njk

```diff
@@ -1,10 +1,10 @@
 {% macro navbar(isHomePage = false, env = env) %}
-<nav class="w-full flex justify-center glass-nav {% if isHomePage %}bg-transparent opacity-0 -translate-y-full{% else %}bg-pycon-blue/70{% endif %} z-50 fixed top-0 transition-all duration-500" id="main-navbar">
+<nav class="w-full flex justify-center {% if isHomePage %}bg-black{% else %}bg-pycon-blue{% endif %} z-50 fixed top-0 left-0 right-0 transition-all duration-500" id="main-navbar">
   <div class="w-full max-w-5xl mx-auto px-6 hidden md:block py-3">
     <div class="relative flex justify-center items-center h-12 px-6">
-      <a href="{{ env.baseUrl }}" class="absolute left-0">
+      {# <a href="{{ env.baseUrl }}" class="absolute left-0">
         <img src="{{ env.baseUrl }}img/logo.png" alt="PyCon India 2025" class="h-10 w-auto">
-      </a>
+      </a> #}
```

Comment:
@DevilsAutumn I think we should leave this since it is required to navigate to the main page.

Reply:
+1
Yes don't remove it. It's an acceptable change from the figma.